index.php

```diff
@@ -81,6 +86,7 @@
 
     // Loop through configured display fields and add them.
     foreach ($fields as $field) {
+        $field = trim($field);
```

Comment:
@azito122 We trim here, but then also in `report_roster_process_field`. Are both necessary? `$field` isn't used again.

Reply:
@mackensen Good catch. Removed.